optimize/client/scripts/dependencies.js

```diff
@@ -63,3 +62,15 @@ menu:
 `;
 
 fs.writeFileSync(depMdFile, fileHeader + lines.join('\n'));
+
+function toPairs(obj) {
+  const pairs = [];
+
+  for (const key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      pairs.push([key, obj[key]]);
+    }
+  }
+
+  return pairs;
```

Comment:
Won't this do the same thing?
```suggestion
  return Object.entries(obj);
```